lib/server.js

```diff
@@ -81,16 +82,13 @@ async function initConfiguration(rawConfig) {
     // configure logger
     config.logs = config.logs || {};
     config.logs.logsDir = path.normalize(config.logs.logsDir || 'logs');
-    if (!config.logger) {
-      await fse.ensureDir(config.logs.logsDir);
-      logger.configure(config.logs);
-      config.logger = logger;
-    } else {
-      // TODO: global logger concept probably needs rewrite
-      ({ logger } = config);
-    }
+    await fse.ensureDir(config.logs.logsDir);
+    rootLogger.loggers.set(
+      'git-server-errors-6ae5f55e-dbb3-46a0-a596-c238e713c1cc',
+      new FileLogger(`${config.logs.logsDir}/error.log`, { level: config.logs.level || 'info' }),
```

Comment:
> `'git-server-errors-6ae5f55e-dbb3-46a0-a596-c238e713c1cc'`

what does this UUID refer to?

providing at least inline comments documenting the usage would be nice since `helix-log` doesn't seem to provide any documentation.

Reply:
I have updated the code. Please do take a look at helix-log. It currently provides ample documentation; in fact, almost 35% of the code under src/ is api documentation (which is a pretty good ratio).

Reply:
I looked at the `README` and in the project. API documentation in the source files is great but for a general purpose library like `helix-log` I would expect that the `README` provides documentation and usage examples and/or a link to generated documentation.

Reply:
I think expanding the readme is a great idea and I am happy to take PRs.